scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ForkedTaskVariablesManager.java

```diff
@@ -82,6 +82,8 @@ public void addBindingsToScriptHandler(ScriptHandler scriptHandler, TaskContext
             scriptHandler.addBinding(SchedulerConstants.DS_USER_API_BINDING_NAME, userSpaceClient);
             scriptHandler.addBinding(SchedulerConstants.DS_GLOBAL_API_BINDING_NAME, globalSpaceClient);
         }
+        scriptHandler.addBinding(SchedulerConstants.SYNCHRONIZATIONAPI_BINDING_NAME,
```

Comment:
Note that the other constants use an underscore before "API". To do the same we should rename into `SYNCHRONIZATION_API_BINDING_NAME`.

Reply:
ok